server/api/reports.py

```diff
@@ -72,28 +74,57 @@ def pretty_batch_ticket_numbers(batch: Batch, round_id_to_num: Dict[str, int]) -
     return ", ".join(ticket_numbers)
 
 
-def pretty_ballot_interpretations(
-    interpretations: List[BallotInterpretation], contests: List[Contest],
-) -> List[str]:
-    columns = []
-    for contest in contests:
-        interpretation = next(
-            (i for i in interpretations if i.contest_id == contest.id), None,
-        )
-        if interpretation:
-            choices = (
-                ", ".join(choice.name for choice in interpretation.selected_choices)
-                if interpretation.interpretation == Interpretation.VOTE
-                else interpretation.interpretation
-            )
-            overvote = "OVERVOTE; " if interpretation.is_overvote else ""
-            comment = (
-                f"; COMMENT: {interpretation.comment}" if interpretation.comment else ""
-            )
-            columns.append(overvote + choices + comment)
-        else:
-            columns.append("")
-    return columns
+def pretty_ballot_interpretation(
+    interpretations: List[BallotInterpretation], contest: Contest,
+) -> str:
+    interpretation = next(
+        (i for i in interpretations if i.contest_id == contest.id), None,
+    )
+    if not interpretation:
+        return ""
+
+    choices = (
+        ", ".join(choice.name for choice in interpretation.selected_choices)
+        if interpretation.interpretation == Interpretation.VOTE
+        else interpretation.interpretation
+    )
+    overvote = "OVERVOTE; " if interpretation.is_overvote else ""
+    comment = f"; COMMENT: {interpretation.comment}" if interpretation.comment else ""
+    return overvote + choices + comment
+
+
+def pretty_cvr_interpretation(
+    ballot: SampledBallot, contest: Contest, contest_cvrs: supersimple.CVRS
+) -> str:
+    ballot_key = (
+        list(ballot.draws)[0].ticket_number if contest.is_targeted else ballot.id
+    )
+    cvrs_by_choice = contest_cvrs[ballot_key].get(contest.id)
+
+    # If CVR was empty for this contest for this ballot, skip it
+    if not cvrs_by_choice:
+        return ""
+
+    choice_id_to_name = {choice.id: choice.name for choice in contest.choices}
+    return ", ".join(
+        choice_id_to_name[choice_id]
+        for choice_id, interpretation in cvrs_by_choice.items()
+        if interpretation == 1
+    )
+
+
+def pretty_discrepancy(
+    ballot: SampledBallot,
+    contest: Contest,
+    contest_discrepancies: Dict[str, supersimple.Discrepancy],
+) -> str:
+    ballot_key = (
+        list(ballot.draws)[0].ticket_number if contest.is_targeted else ballot.id
+    )
+    if ballot_key in contest_discrepancies:
+        return str(contest_discrepancies[ballot_key]["counted_as"])
```

Comment:
Just to be clear, this is just converting an integer to a string. Is that the intended behavior?

Reply:
yeah cuz we are printing it in the audit report!

Reply:
I'm just trying to clarify, this will literally just be like "2" in the discrepancies field. Do we want something a little more information-dense than that? @mcchilders?

Reply:
ah good question. i think in the context of the report it's all the info you need, because there are other columns with the CVR/audit result. check out one of the snapshot reports. but def open to feedback